mod/metrics/set/type_plus_right/metric/value_type.rb

```diff
@@ -59,12 +59,16 @@ def default_edit_args _args
     voo.title = "Value Type"
   end
 
-  def multi_card_edit_slot
+  def multi_card_edit
     super() + fields_form
   end
 
-  def single_card_edit_slot
-    multi_card_edit_slot
+  def single_card_edit_field
+    super() + fields_form
+  end
+
+  def editor_in_multi_card
+    super() + fields_form
```

Comment:
value type editor was working for the last four months because these methods were renamed in decko.

Reply:
very hacky, we need a clearer way to add additional stuff to all kind of editors. Probably a edit structure, but using self in the edit structure needs other hacky stuff to prevent loops.